src/main/java/ai/rev/languageid/LanguageIdClient.java

```diff
@@ -0,0 +1,313 @@
+package ai.rev.languageid;
+
+import ai.rev.helpers.ClientHelper;
+import ai.rev.languageid.models.LanguageIdJob;
+import ai.rev.languageid.models.LanguageIdJobOptions;
+import ai.rev.languageid.models.LanguageIdResult;
+import ai.rev.speechtotext.FileStreamRequestBody;
+import okhttp3.MediaType;
+import okhttp3.MultipartBody;
+import okhttp3.OkHttpClient;
+import okhttp3.RequestBody;
+import retrofit2.Retrofit;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * The LanguageIdClient object provides methods to send and retrieve information from all the
+ * Rev AI Language Identification API endpoints using the Retrofit HTTP client.
+ */
+public class LanguageIdClient {
+
+    private OkHttpClient client;
+
+    /**
+     * Interface that LanguageIdClient methods use to make requests
+     */
+    public LanguageIdInterface apiInterface;
+
+    /**
+     * Constructs the API client used to send HTTP requests to Rev AI. The user access token can be
+     * generated on the website at <a
+     * href="https://www.rev.ai/access_token">https://www.rev.ai/access_token</a>.
+     *
+     * @param accessToken Rev AI authorization token associate with the account.
+     * @throws IllegalArgumentException If the access token is null or empty.
+     */
+    public LanguageIdClient(String accessToken) {
+        if (accessToken == null || accessToken.isEmpty()) {
+            throw new IllegalArgumentException("Access token must be provided");
+        }
+        this.client = ClientHelper.createOkHttpClient(accessToken);
+        Retrofit retrofit = ClientHelper.createRetrofitInstance(client, "languageid", "v1");
+        this.apiInterface = retrofit.create(LanguageIdInterface.class);
+    }
+
+    /**
+     * Manually closes the connection when the code is running in a JVM
+     */
+    public void closeConnection() {
+        client.dispatcher().executorService().shutdown();
+        client.connectionPool().evictAll();
+    }
+
+    /**
+     * This method sends a GET request to the /jobs endpoint and returns a list of {@link LanguageIdJob}
+     * objects.
+     *
+     * @param limit         The maximum number of jobs to return. The default is 100, max is 1000.
```

Comment:
This is indented to match the other param here but you don't do this elsewhere, I'm not sure which is right but we should be consistent